ddtrace/tracer/textmap.go

```diff
@@ -95,6 +95,12 @@ const originHeader = "x-datadog-origin"
 // traceTagsHeader holds the propagated trace tags
 const traceTagsHeader = "x-datadog-tags"
 
+// propagationErrorKey holds any error from propagated trace tags (if any)
+const propagationErrorKey = "_dd.propagation_error"
```

Comment:
Can we group them with the rest of the bunch? (and maybe use the same naming style)

https://github.com/DataDog/dd-trace-go/blob/bd20964575c07f00d616972129c540f47b5dcd81/ddtrace/tracer/span.go#L563-L573